linux/src/sync_linux.c

```diff
@@ -14,11 +14,11 @@ IMPLEMENT_MOCKABLE_FUNCTION(, bool, wait_on_address, volatile_atomic int32_t*, a
     (void) timeout_ms;
     return false;

```

Comment:
will probably need an implementation to get the unit tests to pass :)

Reply:
Yeah I forgot about that, coming up in next iteration

---
In reply to: [443819576](https://github.com/Azure/azure-c-pal/pull/6#discussion_r443819576) [](ancestors = 443819576)